man/fi_endpoint.3.md

```diff
@@ -513,6 +513,9 @@ The following option levels and option names and parameters are defined.
   sized renedezvous protocol message usually results in better latency for the
   overall transfer of a large message.
 
+- *FI_OPT_TRAFFIC_CLASS - uint32_t*
+: Defines the traffic class used for transactions initiated on the endpoint.
+  The provider maps an endpoints' traffic class to an available traffic class.
```

Comment:
This doesn't change the ABI, so could be added later if it's needed.  For now, I would drop this option and require that the tclass be specified at EP creation time through the Tx attributes.  Keeping this option means defining when the tclass can be set/changed.  To support more implementations, I prefer to restrict when the tclass is set, assuming that apps really don't need to change it while an EP is active.

Reply:
Removing it will not allow an application to use an EP alias with a different TC value since the alias will inherit the tx context from the original EP. Do we want to do that with this initial commit?

Reply:
An EP alias is intended to provide an optimized software path, not an alternative HW configuration.  I would vote against letting an EP alias define a different TC from the actual EP.  An implementation may embed the TC as part of the HW (transport offload)

Reply:
I don't see it as a hardware configuration aspect at this point, but I agree that we could defer it for now.